travis/update-release.py

```diff
@@ -230,12 +265,17 @@ def count_severity(filename):
 
 
 else:
-    if len(sys.argv) != 3:
-        print("Usage: python update-release.py PYPI_REGISTRY TAG_NAME")
+    if len(sys.argv) != 4:
+        print("Usage: python update-release.py PYPI_REGISTRY TAG_NAME RELEASE")
         sys.exit(1)
 
     PYPI_REGISTRY = sys.argv[1]
     TAG_NAME = sys.argv[2]
+    RELEASE = sys.argv[3]
+    
+    if RELEASE == "True":
+        # remove .z from RELEASE_TAG
+        RELEASE_TAG = RELEASE_TAG[:-2]
```

Comment:
so every time in code we access RELEASE_TAG via os.get_enviro so make sure you do it once and reflects at all places

Reply:
sure, I'm adding a `.z` to the release tag at the start of the script, when we initially get it via os.get_enviro.
in the acc-provision block, I'm checking if it's a release to remove the `.z` from the release_name.